Detectors/MUON/MCH/Simulation/src/Response.cxx

```diff
@@ -86,6 +99,13 @@ double Response::chargefrac1d(float min, float max, double k2, double sqrtk3, do
 double Response::response(float charge)
 {
   //FEE effects
+  if (charge < mChargeThreshold)
```

Comment:
formatting remark here : you should but braces for the ifs even for one liners (https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Braces#Braces)